README.md

```diff
@@ -4,4 +4,8 @@ Welcome to the [SPARQL-dev Community Group](https://www.w3.org/community/sparql-
 
 [Mailing list archive](https://lists.w3.org/Archives/Public/public-sparql-dev/).
 
-There are a lot of [issues](https://github.com/w3c/sparql-dev/issues) being worked on and also have a look at the [wiki](https://github.com/w3c/sparql-dev/wiki)
+There are a lot of [issues](https://github.com/w3c/sparql-dev/issues) and being
```

Comment:
I am confused by the "and being worked on" but it's maybe my english that is failing me.

Reply:
Fixed - thanks.